artist/src/main/kotlin/com/uber/artist/Artist.kt

```diff
@@ -107,9 +107,6 @@ private fun createInitBuilderFor(
         rClass: ClassName): MethodSpec.Builder {
     val initMethod = MethodSpec.methodBuilder("init")
             .addAnnotation(TypeNames.Annotations.CallSuper)
-            .addAnnotation(AnnotationSpec.builder(SuppressWarnings::class.java)
```

Comment:
it might be a good idea to move this to a `SuppressNullabilityInitializerTrait` in the `artist-traits` module in case others are leveraging this behavior